packages/mobile/src/components/CurrencyDisplay.tsx

```diff
@@ -2,61 +2,128 @@ import colors from '@celo/react-components/styles/colors'
 import fontStyles from '@celo/react-components/styles/fonts'
 import BigNumber from 'bignumber.js'
 import * as React from 'react'
-import { StyleSheet, Text, View } from 'react-native'
+import { StyleProp, StyleSheet, Text, TextStyle, View } from 'react-native'
+import { MoneyAmount } from 'src/apollo/types'
 import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
-import { LocalCurrencySymbol } from 'src/localCurrency/consts'
+import { LocalCurrencyCode, LocalCurrencySymbol } from 'src/localCurrency/consts'
+import { useDollarsToLocalAmount, useLocalCurrencyCode } from 'src/localCurrency/hooks'
 import { getMoneyDisplayValue } from 'src/utils/formatting'
 
+export enum DisplayType {
+  Default,
+  Big, // symbol displayed as superscript
+}
+
 interface Props {
-  amount: BigNumber
-  size: number
-  type: CURRENCY_ENUM
-  currencySymbol?: LocalCurrencySymbol | null
+  type: DisplayType
+  amount: MoneyAmount
+  size: number // only used for DisplayType.Big
```

Comment:
what is size?

Reply:
It's the fontSize, somewhat redundant with the style property. I want to address that in a follow up PR.